cellxgene_schema_cli/cellxgene_schema/migrate.py

```diff
@@ -181,11 +200,135 @@ def migrate(input_file, output_file, collection_id, dataset_id):
+    '''
 
-    # AUTOMATED, DO NOT CHANGE -- IF GENCODE UPDATED, DEPRECATED FEATURE FILTERING ALGORITHM WILL GO HERE.
-    if DEPRECATED_FEATURE_IDS:
-        dataset = utils.remove_deprecated_features(adata=dataset, deprecated=DEPRECATED_FEATURE_IDS)
+    # Schema 5.3 Migration
+    # https://github.com/chanzuckerberg/single-cell-curation/issues/1023
+
+    # Checking pre-identified datasets with non-csr matrices for sparsity
+    # If sparsity is high enough, converting to sparse_csr
+    if dataset_id in non_csr_list:
+        dataset = dataset.to_memory()
+        dataset = utils.check_non_csr_matrices(dataset)
+
+    # Same check, but for private datasets
+    if dataset.uns['title'] in list(zip(*private_non_csr_list))[0]:
+        dataset_donors = dataset.obs['donor_id'].unique()
+        for d in dataset_donors:
+            if (dataset.uns["title"],d) in private_non_csr_list:
+                dataset = dataset.to_memory()
+                dataset = utils.check_non_csr_matrices(dataset)
+
+    # Public dataset changes
+    if collection_id == "33f48a52-31d8-4cc8-bd00-1e89c659a87f":
+        utils.replace_ontology_term(dataset.obs, "cell_type", {"CL:0000065": "CL:4052001"})
+        utils.replace_ontology_term(dataset.obs, "tissue", {"CL:0000065":"CL:4052001"})
+
+    if collection_id == "ef9f0fae-c12c-4f1a-bb67-63346c2ae0f1":
+        utils.map_ontology_term(dataset.obs,
+         "cell_type",
+         "author_cell_type", 
+         {
+            "Type IIX": "CL:4052026",
+            "NMJ":"CL:4052025",
+            "MRJ":"CL:4052024"
+         })
+
+    if collection_id == "964581c3-b5ac-494d-9bd4-0dcb6fb058da":
+        utils.map_ontology_term(dataset.obs,
+         "cell_type",
+         "author_cell_type", 
+         {
+            "Intercalated": "CL:4052048",
+            "Striated": "CL:4052049"
+         })
+
+    if collection_id == "350fe77e-cc64-482e-a42a-18bec9e4a629":
+        utils.map_ontology_term(dataset.obs, "cell_type", "cluster.names", {"Early Theca": "CL:4052010"})
+
+    if collection_id == "1e313b15-4aca-4e5a-94a3-1093c4c42abd":
+        utils.replace_ontology_term(dataset.obs, "assay", {"EFO:0008953":"EFO:0022845"})
+
+    if collection_id == "4c6eaf5c-6d57-4c76-b1e9-60df8c655f1e":
+        utils.map_ontology_term(dataset.obs,
+         "development_stage",
+         "donor_id", 
+         {
+            "MMD_22_20137": "HsapDv:0000274",
+            "A23_5566": "HsapDv:0000274",
+            "19-D011": "HsapDv:0000274",
+            "D018-13": "HsapDv:0000274",
+            "19-D006": "HsapDv:0000274"
+         })
+
+    if collection_id == "8f126edf-5405-4731-8374-b5ce11f53e82":
+        utils.map_ontology_term(dataset.obs,
+         "development_stage",
+         "donor_id", 
+         {
+            "S00006": "HsapDv:0000274",
+            "S00063": "HsapDv:0000274"
+         })
+
+    # Private dataset changes
+
+    # Changes for dataset in private collection "Mouse Post-Flu Time Series"
```

Comment:
The "Mouse Post-Flu Time Series" has since been published: https://cellxgene.cziscience.com/collections/606dc65c-ebd8-4f93-ac86-d0789c04863f